### DISCUSSION TITLE
Tabler logo.svg font name

### DISCUSSION THREAD

**Discussion by electrocoder** [tabler/tabler#1286] [General]
Hi, what is the name of the svg logo font.
![image](https://user-images.githubusercontent.com/628144/192320482-8f9d0230-1a7c-42ec-b48d-d8daf868cb7d.png)

**Comment by codecalm** [comment]
it's Nunito font :)

https://fonts.google.com/specimen/Nunito?query=nunito

**Reply by electrocoder** [reply]
Thanks

**Reply by thesuhu** [reply]
It's looks like Quicksand Bold.

<img width="430" alt="image" src="https://user-images.githubusercontent.com/1247617/198681438-e952f678-0b5c-4227-98f5-8263f0b6099f.png">

**Reply by codecalm** [reply]
damn, my bad. It's of course Quicksand Bold, Nunito was used in previous version in body text

**Reply by ZeoNish** [reply]
thx :-)
